.github/workflows/pages.yml

```
@@ -0,0 +1,54 @@
+      name: github-pages
+      url: ${{ steps.deployment.outputs.page_url }}
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - name: Setup Pages
+        uses: actions/configure-pages@v5
+        with:
+          # Explicitly disable Jekyll
+          enabler_jekyll: false
```

actionlint (1.7.10)

input "enabler_jekyll" is not defined in action "actions/configure-pages@v5". available inputs are "enablement", "generator_config_file", "static_site_generator", "token"

(action)